services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/ControllerMetricEntity.java

```diff
@@ -15,34 +16,52 @@
 
 
 public enum ControllerMetricEntity implements ModuleMetricEntityInterface {
-  REPUSH_CALL_COUNT(
-      MetricType.COUNTER, MetricUnit.NUMBER, "Count of all calls to a controller endpoint",
-      setOf(VENICE_STORE_NAME, VENICE_RESPONSE_STATUS_CODE_CATEGORY, VENICE_CLUSTER_NAME, REPUSH_TRIGGER_SOURCE)
+  /** Count of all requests to repush a store */
+  STORE_REPUSH_CALL_COUNT(
+      "store.repush.call_count", MetricType.COUNTER, MetricUnit.NUMBER, "Count of all requests to repush a store",
+      setOf(VENICE_STORE_NAME, VENICE_RESPONSE_STATUS_CODE_CATEGORY, VENICE_CLUSTER_NAME, STORE_REPUSH_TRIGGER_SOURCE)
   ),
-  COMPACTION_ELIGIBLE_STATE(
-      MetricType.GAUGE, MetricUnit.NUMBER,
-      "This metric indicates the duration from when a store is first nominated for compaction until the store is compacted successfully."
-          + " When a store is nominated for scheduled compaction and remains uncompacted, this metric will be at 1."
-          + " When the store is compacted, this metric will return to 0.",
+
+  /** log compaction related metrics */
+
+  /** Count of stores nominated for scheduled compaction */
+  STORE_COMPACTION_NOMINATED_COUNT(
+      "store.compaction.nominated_count", MetricType.COUNTER, MetricUnit.NUMBER,
+      "Count of stores nominated for scheduled compaction", setOf(VENICE_STORE_NAME, VENICE_CLUSTER_NAME)
+  ),
+
+  /**
+   * Track the state from the time a store is nominated for compaction to the
+   * time the repush is completed to finish compaction. stays 1 after nomination
+   * and becomes 0 when the compaction is compacted
+   */
+  STORE_COMPACTION_ELIGIBLE_STATE(
+      "store.compaction.eligible_state", MetricType.GAUGE, MetricUnit.NUMBER,
+      "Track the state from the time a store is nominated for compaction to the time the repush is completed",
       setOf(VENICE_STORE_NAME, VENICE_CLUSTER_NAME)
   ),
-  STORE_NOMINATED_FOR_COMPACTION_COUNT(
-      MetricType.COUNTER, MetricUnit.NUMBER, "Count of stores nominated for scheduled compaction",
+
+  /** Count of log compaction repush triggered for a store after it becomes eligible */
+  STORE_COMPACTION_TRIGGERED_COUNT(
```

Comment:
I don't seem to find the usage of this metric `STORE_COMPACTION_TRIGGERED_COUNT` in this change. Is it prepared for later?

Reply:
Good catch, added it. thanks.